### ISSUE TITLE
DNN 9.2 Compatibility - v01.04.01

### ISSUE THREAD

**Issue by birksy89** [2sic/app-mobius-forms#10]
Hello, does this module work in DNN 9.2

I've just installed the latest version of 2sxc content (9.23.0)
And Mobius Forms (v01.04.01)

I get the below error - It seems to be referencing a path that isn't on my system.

Please let me know if there's any way I can resolve this.

Thanks in advance

`System.Web.Compilation.BuildManager.GetCompiledType(VirtualPath virtualPath) at ToSic.SexyContent.Engines.RazorEngine.CreateWebPageInstance() in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2Sexy Content Razor\RazorEngine.cs:line 109 at ToSic.SexyContent.Engines.RazorEngine.InitWebpage() in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2Sexy Content Razor\RazorEngine.cs:line 126 at ToSic.SexyContent.Engines.RazorEngine.Init() in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2Sexy Content Razor\RazorEngine.cs:line 30 at ToSic.SexyContent.Engines.EngineBase.Init(Template template, App app, IInstanceInfo hostingModule, IDataSource dataSource, InstancePurposes instancePurposes, SxcInstance sxcInstance, Log parentLog) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\ToSic.Sxc\SexyContent\Engines\EngineBase.cs:line 62 at ToSic.SexyContent.SxcInstance.GetRenderingEngine(InstancePurposes renderingPurpose) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\ToSic.Sxc\SexyContent\SxcInstance_Render.cs:line 121 at ToSic.SexyContent.SxcInstance.Render() in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\ToSic.Sxc\SexyContent\SxcInstance_Render.cs:line 29`

> at System.ThrowHelper.ThrowKeyNotFoundException() at System.Collections.Generic.Dictionary`2.get_Item(TKey key) at ToSic.Eav.Apps.Template.get_Query() at ToSic.SexyContent.Edit.ClientContextInfo.ClientInfoContentGroup..ctor(SxcInstance sxc, Boolean isCreated) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\ToSic.Sxc\Edit\ClientContextInfo\ClientInfoContentGroup.cs:line 31 at ToSic.SexyContent.Environment.Dnn7.ClientInfosAll..ctor(String systemRootUrl, PortalSettings ps, IInstanceInfo mic, SxcInstance sxc, UserInfo uinfo, Int32 zoneId, Boolean isCreated, Boolean autoToolbar, Log parentLog) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2sxc Dnn\Environment\Dnn7\ClientInfos\ClientInfosAll.cs:line 34 at ToSic.SexyContent.Environment.Dnn7.DnnRenderingHelpers.UiContextInfos(Boolean autoToolbars) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2sxc Dnn\Environment\Dnn7\DnnRenderingHelpers.cs:line 123 at ToSic.SexyContent.Environment.Dnn7.DnnRenderingHelpers.ContextAttributes(Int32 instanceId, Int32 contentBlockId, Boolean includeEditInfos, Boolean autoToolbar) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2sxc Dnn\Environment\Dnn7\DnnRenderingHelpers.cs:line 78 at ToSic.SexyContent.Environment.Dnn7.DnnRenderingHelpers.WrapInContext(String content, String dontRelyOnParameterOrder, Int32 instanceId, Int32 contentBlockId, Boolean editContext, String tag, Boolean autoToolbar, Boolean addLineBreaks) in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\2sxc Dnn\Environment\Dnn7\DnnRenderingHelpers.cs:line 58 at ToSic.SexyContent.SxcInstance.Render() in C:\Projects\2sxc-dnn742\Website\DesktopModules\ToSIC_SexyContent\ToSic.Sxc\SexyContent\SxcInstance_Render.cs:line 76

**Comment by iJungleboy** [comment]
I don't have a quick answer - could you enable extensive logging and post that? https://2sxc.org/en/blog/post/releasing-2sxc-9-6-with-extensive-logging

**Comment by birksy89** [comment]
Hi Daniel,

When I've gone to restore this - The original error wasn't there.

However, I did struggle to get it working. One issue was that the SMTP settings that I'm using (Google) requires SSL.

I've altered this in the FormController.cs but this might be suited to being a setting via 2sxc.

![capture](https://user-images.githubusercontent.com/4971630/39125792-2793a9c2-46f8-11e8-9f10-03d1f05bad6e.PNG)

This current issue can be closed - Sorry!
